jsx/App/Stories/Story/Display/Latex/TierButtonList.jsx

```diff
@@ -10,43 +10,27 @@ export const TierButtonList = ({ sentenceId, tierNames, latexSectionId, latexSec
   return (
     <div className="tierSelectionRow">
       <b><TranslatableText dictionary={latexSectionName} /></b>
-      <TierRadioButtons sentenceId={sentenceId} tierNames={tierNames} latexSectionId={latexSectionId} latexSectionName={latexSectionName} />
+      <div className="tierSelectionRowButtonSection">
+        {tierNames.map((tierName, i) => {
+          return (<TierRadioButton 
```

Comment:
Nice! I haven't looked at the visuals yet, but mapping each tier to a `TierRadioButton` is definitely more elegant than having a plural `TierRadioButtons` component. Thanks for doing this work! :)